.github/workflows/ci.yml

```diff
@@ -895,8 +900,10 @@ jobs:
         python-version: ${{ matrix.python }}
 
     - name: Prepare env
+      # Ensure use of NumPy 2 (via NumPy nightlies but can be changed soon)
       run: |
         python3 -m pip install -r tests/requirements.txt
+        python3 -m pip install --upgrade --pre -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy scipy
```

Comment:
I think both are on regular PyPI now.

Reply:
Right, plus I keep forgetting that `>=2.0.0b1` works well, so changed.  Changed.